site/components/pages/Pledge/lib/verifyGnosisSafeMultisig.ts

```diff
@@ -0,0 +1,32 @@
+import { ethers } from "ethers";
+import { getJsonRpcProvider } from "@klimadao/lib/utils";
+import GnosisSafeSignMessageLib from "@klimadao/lib/abi/GnosisSafeSignMessageLib.json";
+
+const ONE_HOUR = 60 * 60 * 1000;
+
+export const verifyGnosisSafeMultisig = async (params: {
+  signature: string;
```

Comment:
I realized the naming here can be confusing. This is not actually a signature, it's what the ethers docs call a "message" (unsigned string)